python/shotgun_fields/date_and_time_widget.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Widget that represents the value of a date_time field in Shotgun
+"""
+import datetime
+from .label_base_widget import LabelBaseWidget
+from .widget_metaclass import ShotgunFieldMeta
+
+
+class DateAndTimeWidget(LabelBaseWidget):
+    """
+    Inherited from a :class:`~LabelBaseWidget`, this class is able to
+    display a date_time field value as returned by the Shotgun API.
+    """
+    __metaclass__ = ShotgunFieldMeta
+    _FIELD_TYPE = "date_time"
+
+    def _string_value(self, value):
+        """
+        Convert the Shotgun value for this field into a string
+
+        :param value: The value to convert into a string
+        :type value: :class:`datetime.datetime`
+        """
+        return self._create_human_readable_timestamp(value, " %I:%M%p")
```

Comment:
Looking here: https://docs.python.org/2/library/time.html#time.strftime
Could `%X` be used instead so that the default locale is taken int account?